portfolio/src/main/webapp/script.js

```diff
@@ -140,6 +137,37 @@ function createListElement(msg) {
   return postElement;
 }
 
+// Creates a list of posts to be displayed on the homepage.
+// Display-only.(ie. No reply or delete button). 
+function createListElementHome(msg) {
+  const postElement = document.createElement('li');
+  postElement.style.margin = "10px";
```

Comment:
It's probably better to do this in a css file and give your element a class to control the styling. That way your styling and processing logic are separate.